lib/mimdb/movies/role.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Mimdb.Movies.Role do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "roles" do
+    field :character, :string
+    field :movie_id, :id
+    field :actor_id, :id
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(role, attrs) do
+    role
+    |> cast(attrs, [:character, :actor_id, :movie_id])
+    |> validate_required([:character])
```

Comment:
I think we should require both id's as well

Reply:
Requiring both ids now